hack/install_bats.sh

```diff
@@ -4,9 +4,9 @@ set -e
 
 die() { echo "${1:-No error message given} (from $(basename $0))"; exit 1; }
 
-if [[ "$(type -t bats)" != "" ]]; then
-	# bats is already installed.
-	exit 0
+if [[ -n "$(type -P bats)" ]]; then
```

Comment:
ahh.  Well, hmmm, damn.  I don't want to fight with `$PATH` :disappointed: 

@edsantiago is it safe to assume the system tests are/should started by the `Makefile`?  If so, then maybe we just hard-code it to use `/usr/local/bin/bats`?

Another idea: We strip out the target and install script (since Makefiles really shouldn't do this kind of thing anyway) and just modify the VM images to install the bats packages.  IIRC, we originally locked-in on 1.1.0 due to some `exec` bug.  I'm assuming that's fixed in all modern bats versions.